storage-resize-images/POSTINSTALL.md

```diff
@@ -4,7 +4,7 @@ You can test out this extension right away:
 
 1.  Go to your [Storage dashboard](https://console.firebase.google.com/project/${param:PROJECT_ID}/storage).
 
-1.  Upload an image file to the bucket: `${param:IMG_BUCKET}`
+1.  Upload an image file to the bucket: `${param:IMG_BUCKET}`. The file must have a valid [image MIME Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types#Image_types), for example `image/png`.
```

Comment:
Is this a really common case that a dev might not have the image MIME type specified?  If not, then I wouldn't put this level of detail here in this uber quickstart section.  Instead, I'd like to explore putting some of these known gotchas within the "Using this extension" section.

Reply:
Yes, agreed.

Reply:
Moved this "error" down in the POSTINSTALL file.  Pls double-check that my change and additional text is correct?  Thx!

Reply:
ping on this comment thread :-)

Reply:
Looks good to me.

Reply:
@laurenzlong Are you ok with the changes?